x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/ThrowingDriverContext.java

```diff
@@ -18,7 +19,7 @@
 
 public class ThrowingDriverContext extends DriverContext {
     public ThrowingDriverContext() {
-        super(new ThrowingBigArrays(), BlockFactory.getNonBreakingInstance());
```

Comment:
out-of-scope: I was confused whether we need to follow this one up as well or if its okay to use a noop circuit breaker. Can we add a short javadoc describing the purpose of `ThrowingDriverContext`? Turns out it's only used in `FilterOperatorFactory.describe`.

In fact, knowing this it would be more correct to use a circuit breaker that also always throws if we try to request memory from it.

I can add a javadoc as well as this remark is not directly to your PR's contents.